packages/preview/create-lattes-cv/0.1.0/lib.typ

```diff
@@ -0,0 +1,140 @@
+#import "@preview/datify:0.1.3": *
+#import "src/import.typ": *
+
+// Função lattes_cv: criar o PDF com os dados de lattes
+// Arguments:
+// - database: o arquivo de TOML com os dados de Lattes (string)
+// - kind: o tipo de currículo Lattes (string)
+// - me: o nome para destacar nas citações (string)
+// - last_page: resumo de produção no final (boolean)
+// - data: a data de currículo (datetime)
+// - subtitle: para a página inicial 
+#let lattes-cv(
+  database: "output.toml",
+  kind: "completo", 
+  me: str,
+  last_page: true,
```

Comment:
As suggested by the package review bot, it would be better if this parameter could use `kebab-case` instead of `snake_case`. And there are a few other occurrences of this in your package.